tests/qt_types_standalone/cpp/qfontmetrics.h

```diff
@@ -0,0 +1,34 @@
+// clang-format off
+// SPDX-FileCopyrightText: 2024 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// clang-format on
+// SPDX-FileContributor: Laurent Montel <laurent.montel@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+#pragma once
+
+#include <QtGui/QFont>
+#include <QtGui/QFontMetrics>
+#include <QtTest/QTest>
+
+#include "cxx-qt-gen/qfontmetrics.cxx.h"
+
+class QFontMetricsTest : public QObject
+{
+  Q_OBJECT
+
+private Q_SLOTS:
+  void construct()
```

Comment:
clang-format is complaining around here

Reply:
Could you use the From to construct in Rust here? eg

- Create a QFont (either C++ or Rust)
- Construct a QFontMetric from the QFont in Rust
- Then check the values are correct in C++